modules/ROOT/pages/indexes/syntax.adoc

```diff
@@ -207,7 +205,7 @@ ON (r.propertyName)
 [OPTIONS “{“ option: value[, …] “}”]
 ----
 
-As of Neo4j 5.18, vector indexes have two vector index providers available, `vector-2.0` (default) and `vector-1.0`.
+As of Neo4j 5.18, vector indexes have two vector index providers available, `vector-2.0` (default) and `vector-1.0` (deprecated).
```

Comment:
why are we leaving this for vector when we just removed it from the others? Wouldn't it make more sense to also remove it from here?

Reply:
since you can't set it anyway I don't see why we'd talk about it here anymore

Reply:
For the vector indexes there is some info about the differences capabilities depending on the providers. Since you can still have indexes on both (just not create new ones) I kept it around for that part to still make sense.
Might not be necessary in this place though. I'll leave it up to Jens to decide what makes most sense

Reply:
I don't think it makes sense in the syntax part, feels like it makes more sense to only have it in the section about the indexes 🤷

Reply:
Let's get rid of it here, and then we can keep the section in the main page about vector indexes.